frontend/src/client/models.ts

```diff
@@ -34,6 +34,8 @@ export type ItemsPublic = {
 };
 
 export interface InquiryCreate {
+  id?: string | null
+  title?: string | null
```

Comment:
title? why title? Is this referenced in the design docs?

Reply:
removed